src/main/java/ua/strongBody/validation/Validator.java

```diff
@@ -3,5 +3,5 @@
 import ua.strongBody.exceptions.ValidationException;
 
 public interface Validator<T, V> {
-    T validate(V v) throws ValidationException;
+    void validate(V v) throws ValidationException;
 }
```

Comment:
If you delete return type, you should delete T from generic.

Reply:
Done